bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/AsterismType.java

```diff
@@ -44,7 +64,12 @@ public Asterism createEmptyAsterism() {
 
         switch (instType) {
             case INSTRUMENT_GHOST:
-                return DefaultImList.create(GhostStandardResolution, GhostHighResolution);
+                return DefaultImList.create(
```

Comment:
There's now an `Instrument` enum.  We could make `instType` be an `Instrument` instead of an `SPComponentType` and then you wouldn't have to check whether it is valid.

Reply:
I think that this would make it difficult when we do the P2 check, since we have the `SpComponentType` of the instrument, but I don't think we'd have the `Instrument` enum value readily available?

The only time this code will be likely be used is for P2 checks, I think. (At least, that's the only time we're using it now.)